openshift/kustomize/services/indexing/base/config-map.yaml

```diff
@@ -24,3 +24,4 @@ data:
   NOTIFICATION_TOPIC: notify
   CHES_EMAIL_ENABLED: "true"
   CHES_EMAIL_AUTHORIZED: "true"
+  INDEX_ONLY: "false"
```

Comment:
The Indexing Service can be configured to only index content.  By default it will update content status.